pkg/cloud/gcp/kms_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 The Cockroach Authors.
```

Comment:
this file looks identical to `amazon/kms_test.go`, how would you feel about doing the following:

- Create a new file in the top level `cloud` pkg called `kms_test_utils.go`.
- Add this code to that file.
- Change `testEncryptDecrypt` to be an exported method called `KMSEncryptDecrypt` or something of the sort, and then invoke that from both `gcp` and `amazon`.
- You might also need to export `testKMSEnv` but that seems fine since theyre shared across all KMS implementers.

Reply:
The only difference is that one calls close and the other doesn't but yeah your solution seems cleaner.